app/scripts/modules/titus/src/serverGroup/details/serverGroupDetails.titus.controller.js

```diff
@@ -108,6 +108,11 @@ angular
           delete labels['source'];
           delete labels['spinnakerAccount'];
 
+          // Server groups with no instances have an empty key/value pair that causes validation issues in the clone modal
+          if (labels[''] === '') {
```

Comment:
is there any reason not to always delete the empty key (i.e., is there a reason to check that the value is also an empty string)?

Reply:
No, we should always delete this, so I am going to update this code